cmake/UserverGrpcTargets.cmake

```diff
@@ -52,7 +52,13 @@ function(_userver_prepare_grpc)
     elseif(protobuf_INCLUDE_DIRS)
         set_property(GLOBAL PROPERTY userver_protobuf_import_dir "${protobuf_INCLUDE_DIRS}")
     else()
-        message(FATAL_ERROR "Invalid Protobuf package")
+        # maybe we found protobuf config
+        get_target_property(_incs protobuf::libprotobuf INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
We can, in fact, remove all the attempts above, this way should work reliably

* We already require `protobuf::libprotobuf` target in `grpc/CMakeLists.txt`
* It seems that Protobuf is committed to putting common protos next to its headers